lib/types/webrtc.d.ts

```diff
@@ -92,6 +93,5 @@ export type RTPEncodingParameters = {
     scaleResolutionDownBy: number;
 };
 export type RTCTrackOptions = {
-    codec?: RTCRtpCodecCapability;
```

Comment:
We are removing codec handling responsibilities from the upper layer (e.g., [client wrapper](https://github.com/mattermost/mattermost-plugin-calls/blob/main/webapp/src/client.ts)). I think that was a mistake as it complicated things further. Most of the complexity of dealing with WebRTC details should be on this side now.